pages/financial.py

```diff
@@ -0,0 +1,287 @@
+st.markdown("""
+Analyzing medical claims by service category allows healthcare insurers to identify patterns, trends, and cost drivers in the service type being performed for the patient.
+
+Here we see that, **outpatient and inpatient spend** have the highest amount of variation over time. Overall spend
+seems to **spike in 2018**, driven by a large increase in outpatient spend 1 month and inpatient
+spend the next.
+""")
+service_1_data = pmpm_by_service_category_1()
+service_1_data = service_1_data.loc[
+    service_1_data["year_month"].str[:4].isin(selected_range)
+]
+
+service_1_chart = alt.Chart(service_1_data).mark_bar().encode(
+    x="year_month",
+    y=alt.Y("paid_amount_pmpm"),
+    color="service_category_1:N",
+    tooltip=["service_category_1", "paid_amount_pmpm"]
+).configure_legend(
+  orient='bottom'
+)
+
+st.altair_chart(service_1_chart, use_container_width=True)
+
+## --------------------------------- ##
+## Service Category 2
+## --------------------------------- ##
+st.markdown("""
+Use the following dropdown to get more detail on the service category that interested you.
+""")
+service_cat_options = service_1_data["service_category_1"].drop_duplicates().tolist()
+selected_service_cat = st.selectbox(
+    label="Select a Service Category",
+    options=service_cat_options,
+    label_visibility='collapsed',
+)
+
+service_2_data = pmpm_by_service_category_1_2()
+service_2_data = service_2_data.loc[
+    service_2_data["year_month"].str[:4].isin(selected_range)
+    & service_2_data["service_category_1"].isin([selected_service_cat])
+].drop("service_category_1", axis=1).reset_index(drop=True)
+service_2_data = service_2_data\
+    .groupby("service_category_2", as_index=False)\
+    [["paid_amount_sum", "member_month_count"]].sum()\
+    .assign(paid_amount_pmpm = lambda x: x["paid_amount_sum"] / x["member_month_count"])
+
+
+service_2_chart = alt.Chart(service_2_data).mark_bar().encode(
+    x="paid_amount_pmpm",
+    y=alt.Y("service_category_2", sort="-x"),
+    tooltip=["service_category_2", "paid_amount_pmpm"]
+)
+
+st.altair_chart(service_2_chart, use_container_width=True)
+
+## --------------------------------- ##
+## Chronic Condition
+## --------------------------------- ##
+st.markdown("## Chronic Condition")
+st.markdown("""
+Certain conditions will be more expensive in your population. Here we see that the top driver of spend
+is `Cardiovascular disease`. The second highest driver is `Metabolic Disease`.
+""")
+chronic_condition_data = pmpm_by_chronic_condition()
+chronic_condition_data = chronic_condition_data.loc[
+    chronic_condition_data["year_month"].str[:4].isin(selected_range)
+]
+chronic_condition_data = chronic_condition_data\
+    .groupby("condition_family", as_index=False)\
+    [["medical_paid_amount_sum", "member_month_count"]].sum()\
+    .assign(medical_paid_amount_pmpm = lambda x: x["medical_paid_amount_sum"] / x["member_month_count"])\
+    .round()
+
+st.dataframe(
```

Comment:
Should probably sort these values by the cost, currently sorted by condition alphabetically

Reply:
good call i'll add right now